### ISSUE TITLE
create from multidev ends in error

### ISSUE THREAD

**Issue by labboy0276** [kalabox/kalabox#780]
Hello!

So when I do a kbox create pantheon && select a multidev environment in 0.10.6 meow I get this at the end:

```
info: DOCKER =>  Creating ad hoc container "kalabox/terminus:t0.9.3". name=kalabox_temp-26e94da7-39cd-41dd-a029-39ef20f9eee1, Cmd=[bash], Tty=true, Image=kalabox/terminus:t0.9.3, Binds=[], Env=[KBOX_CODEDIR=code, KALABOX=true, APPNAME=kfp, APPDOMAIN=kfp.kbox, FRAMEWORK=drupal, DOCROOT=/, FILEMOUNT=sites/default/files, DRUPAL_HASH_SALT=e753a613753f29ad2e856a263179513b4adaf23414351b5a1642f1b6f3d8714d, DB_HOST=kfp.kbox, DB_PORT=3306, DB_USER=pantheon, DB_PASSWORD=, DB_NAME=pantheon, PANTHEON_ACCOUNT=john@kalamuna.com, PANTHEON_SITE=UUID, PANTHEON_SITE_NAME=kfp, PANTHEON_ENVIRONMENT=kalabox, PANTHEON_INFRASTRUCTURE_ENVIRONMENT=kalabox, PRESSFLOW_SETTINGS={"databases":{"default":{"default":{"driver":"mysql","prefix":"","database":"pantheon","username":"pantheon","password":"","host":"kfp.kbox","port":3306}}},"conf":{"pressflow_smart_start":true,"pantheon_binding":"kalabox","pantheon_site_uuid":"kalabox","pantheon_environment":"kalabox","pantheon_tier":"kalabox","pantheon_index_host":"solr.kfp.kbox","pantheon_index_port":449,"redis_client_host":"redis.kfp.kbox","redis_client_port":8161,"redis_client_password":"","file_public_path":"sites/default/files","file_private_path":"sites/default/files/private","file_directory_path":"site/default/files","file_temporary_path":"/tmp","file_directory_temp":"/tmp","css_gzip_compression":false,"js_gzip_compression":false,"page_compression":false},"drupal_hash_salt":"e753a613753f29ad2e856a263179513b4adaf23414351b5a1642f1b6f3d8714d","config_directory_name":"config"}, CACHE_HOST=redis.kfp.kbox, CACHE_PORT=8161, CACHE_PASSWORD=, PANTHEON_INDEX_HOST=solr.kfp.kbox, PANTHEON_INDEX_PORT=449, PHP_VERSION=5.3.29, SSH_KEY=pantheon.kalabox.id_rsa, GITUSER=John Ouellet, GITEMAIL=john@kalamuna.com], NetworkMode=bridge, , VolumesFrom=[kb_kfp_data], WorkingDir=/code, Entrypoint=[/bin/sh, -c]
info: DOCKER =>  Ad hoc container created. 597c2fb40c0605670ae83b8609970456c3e0a8f5722f171495f8f4ad6b323107
info: DOCKER =>  Starting ad hoc container "597c2fb40c0605670ae83b8609970456c3e0a8f5722f171495f8f4ad6b323107". Binds=[/Users/john:/ssh, /Users/john/kalabox/apps/kfp:/src], VolumesFrom=[]
info: DOCKER =>  Ad hoc container started. 597c2fb40c0605670ae83b8609970456c3e0a8f5722f171495f8f4ad6b323107
info: DOCKER =>  Removing ad hoc container. 597c2fb40c0605670ae83b8609970456c3e0a8f5722f171495f8f4ad6b323107
Error dispatching event listeners [post-create-app]: Failed after 5 retries. {"max":5,"backoff":500}: Unexpected token m
VError: Failed after 5 retries. {"max":5,"backoff":500}: Unexpected token m
    at /usr/local/bin/lib/promise.js.jx:61:17
From previous event:
    at rec (/usr/local/bin/lib/promise.js.jx:47:13)
    at /usr/local/bin/lib/promise.js.jx:55:20
From previous event:
    at /usr/local/bin/lib/promise.js.jx:54:12
    at process._tickCallback (node.js:812:13)
```

However, I start the app and all is well, thoughts?

**Comment by pirog** [comment]
looks like a JSON parse error. can you get more verbose output?

**Comment by pirog** [comment]
This should be resolved in the 0.11.0 release on Friday
